configure.ac

```diff
@@ -367,12 +369,15 @@ AS_IF([test "x$with_netcdf" != "xno"],
         AC_MSG_CHECKING([if we can compile NetCDF with C++])
         # Note netcdf_c++ needed
         NCLIB=`$NCCONF --libs | sed s/-lnetcdf/-lnetcdf_c++\ -lnetcdf/`
+        LDFLAGS="$LDFLAGS -Wl,-rpath,`nc-config --prefix`/lib"
```

Comment:
I'm  not sure how critical that line is, as the linker flags should be included in NCLIB. I included it here because it was in the master branch